data/json/items/armor.json

```diff
@@ -992,7 +992,7 @@
     "warmth": 25,
     "material_thickness": 2,
     "environmental_protection": 1,
-    "flags": [ "VARSIZE", "OUTER" ]
+    "flags": [ "VARSIZE", "OUTER", "RAINPROOF", "WATERPROOF" ]
```

Comment:
Are you sure this cotton suit is rainproof?

Reply:
Beekeeping suits are very well-sealed and have full coverage, plus the cotton is rather thick. I thought it was okay to rainproof them, but I can remove that if you feel it's too unrealistic.